modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/DeliveryAgent.java

```diff
@@ -243,10 +243,10 @@ public void connected(HttpRoute route, NHttpClientConnection conn) {
     private void tryNextMessage(MessageContext messageContext, HttpRoute route, NHttpClientConnection conn) {
         if (conn != null) {
             try {
-                if (targetConfiguration.isCorrelationLoggingEnabled()) {
+//                if (targetConfiguration.isCorrelationLoggingEnabled()) {
```

Comment:
@fazlan-nazeem : can you please remove commented out lines

Reply:
Fixed